install-dev/upgrade/php/ps_1770_preset_tab_enabled.php

```diff
@@ -28,9 +28,12 @@
  * Preset enabled new column in tabs to true for all (except for disabled modules)
  */
 function ps_1770_preset_tab_enabled() {
+    // Doctrine schema update is launched after this script so we need to, at least, add this field in database manually
+    $result = Db::getInstance()->execute('ALTER TABLE `' . _DB_PREFIX_ . 'tab` ADD enabled TINYINT(1) NOT NULL');
```

Comment:
@jolelievre Don't you have any error if colum already exist ?

Reply:
Sure, but it doesn't exist at this moment Thats the problem because I can't preset it correctly